### PR TITLE
Update `ballerina test`  and few open-api command help text

### PR DESCRIPTION
**Pull request by rdhananjaya**
## Purpose
$subject.

## Check List 
- [ ] Read the [Contributing Guide](https://github.com/ballerina-platform/ballerina-lang/blob/master/CONTRIBUTING.md)
- [ ] Required Balo version update
- [ ] Updated Change Log
- [ ] Checked Tooling Support (#<Issue Number>)
- [ ] Added necessary tests
  - [ ] Unit Tests
  - [ ] Spec Conformance Tests
  - [ ] Integration Tests
  - [ ] Ballerina By Example Tests
- [ ] Increased Test Coverage   
- [ ] Added necessary documentation  
  - [ ] API documentation 
  - [ ] Module documentation in Module.md files
  - [ ] Ballerina By Examples

### CODE REVIEW

File: cli/ballerina-tool/src/main/resources/cli-help/ballerina-test.help [hunk 1]
```diff
@@ -1,105 +1,51 @@
...
-       ballerina test [-lg|--list-groups] [--groups <test_group, ...>]
-           [--disable-groups <test_group, ...>] [--sourceroot <path>]
-           [-e] [--debug <port>] [-c <config_file>]
-           [--exclude-modules <modules, ...>]
-           [--config <config_file>] [-B] [<balfile>|<module-name>]
+       ballerina test [--offline]
+           [--sourceroot <path>]
+           [--experimental][--skip-lock]
+           [--config <config_file>]
+           [<module-name>|-a|--all]
```
Comment (MaryamZi) [inline cli/ballerina-tool/src/main/resources/cli-help/ballerina-test.help]:
```suggestion
           [<module-name> | -a | --all]
```

File: cli/ballerina-tool/src/main/resources/cli-help/ballerina-test.help [hunk 2]
```diff
@@ -1,105 +1,51 @@
 NAME
-       Ballerina test - Run Ballerina tests and display a summary of
-       test results.
+       ballerina test - execute ballerina tests
```
Comment (MaryamZi) [inline cli/ballerina-tool/src/main/resources/cli-help/ballerina-test.help]:
```suggestion
       ballerina test - execute Ballerina tests
```

File: cli/ballerina-tool/src/main/resources/cli-help/ballerina-test.help [hunk 3]
```diff
@@ -1,105 +1,51 @@
...
 
 SYNOPSIS
-       ballerina test [-lg|--list-groups] [--groups <test_group, ...>]
-           [--disable-groups <test_group, ...>] [--sourceroot <path>]
-           [-e] [--debug <port>] [-c <config_file>]
-           [--exclude-modules <modules, ...>]
-           [--config <config_file>] [-B] [<balfile>|<module-name>]
+       ballerina test [--offline]
+           [--sourceroot <path>]
+           [--experimental][--skip-lock]
```
Comment (MaryamZi) [inline cli/ballerina-tool/src/main/resources/cli-help/ballerina-test.help]:
```suggestion
           [--experimental] [--skip-lock]
```
or move `[--skip-lock]` to a new line?

File: cli/ballerina-tool/src/main/resources/cli-help/ballerina-test.help [hunk 4]
```diff
@@ -1,105 +1,51 @@
...
+           [--config <config_file>]
+           [<module-name>|-a|--all]
 
 
 DESCRIPTION
        Ballerina test command allows you to compile and run Ballerina
```
Comment (MaryamZi) [inline cli/ballerina-tool/src/main/resources/cli-help/ballerina-test.help]:
```suggestion
       Compile and run Ballerina
```

File: cli/ballerina-tool/src/main/resources/cli-help/ballerina-test.help [hunk 5]
```diff
@@ -1,105 +1,51 @@
...
        test sources and prints a summary of test results. You can
```
Comment (MaryamZi) [inline cli/ballerina-tool/src/main/resources/cli-help/ballerina-test.help]:
```suggestion
       test sources and print a summary of test results. You can
```

File: cli/ballerina-tool/src/main/resources/cli-help/ballerina-test.help [hunk 6]
```diff
@@ -1,105 +1,51 @@
...
-       execute tests in a single test file, a test module, or an
-       entire Ballerina project.
-    
-       Use ballerina test without any parameters to execute tests for
-       the entire project.
+       execute tests in a test module, or an entire Ballerina project.
```
Comment (MaryamZi) [inline cli/ballerina-tool/src/main/resources/cli-help/ballerina-test.help]:
```suggestion
       execute tests in a module, or an entire Ballerina project.
```

File: cli/ballerina-tool/src/main/resources/cli-help/ballerina-test.help [hunk 7]
```diff
@@ -1,105 +1,51 @@
...
     
        Use ballerina test <module-name> to execute tests within the
```
Comment (MaryamZi) [inline cli/ballerina-tool/src/main/resources/cli-help/ballerina-test.help]:
```suggestion
        Use 'ballerina test <module-name>' to execute tests in the
```

File: cli/ballerina-tool/src/main/resources/cli-help/ballerina-test.help [hunk 8]
```diff
@@ -1,105 +1,51 @@
...
-       specified module.
-    
-       Use ballerina test <balfile> to execute the given Ballerina test
-       file. Note that imports from other modules will not work for
-       file executions. File path can be a relative or absolute.
+       specified module, or -a or -all to execute tests in all modules.
```
Comment (MaryamZi) [inline cli/ballerina-tool/src/main/resources/cli-help/ballerina-test.help]:
```suggestion
       specified module, or '-a' or '-all' to execute tests in all modules.
```

File: cli/ballerina-tool/src/main/resources/cli-help/ballerina-test.help [hunk 9]
```diff
@@ -1,105 +1,51 @@
...
 
 
 OPTIONS
+        --offline
+            In a module build, the dependencies are always checked from
```
Comment (MaryamZi) [inline cli/ballerina-tool/src/main/resources/cli-help/ballerina-test.help]: Shall we make this uniform with https://github.com/ballerina-platform/ballerina-lang/pull/18834/files#diff-fd96dc1ef9c62010f9406cc6373ea3f4R36

Comment (rdhananjaya) [reply cli/ballerina-tool/src/main/resources/cli-help/ballerina-test.help]: I took it form the commands help text, but let's go with your suggestion.

File: cli/ballerina-tool/src/main/resources/cli-help/ballerina-openapi-gen-client.help [hunk 10]
```diff
@@ -1,12 +1,16 @@
...
+       ballerina openapi gen-client - generate a mock Ballerina client for a given OpenAPI contract
 
-       Note: This is an Experimental tool ship under ballerina hence this will only support limited set of functionality.
 
 SYNOPSIS
        ballerina openapi gen-client [<module>:]<client-name> <openapi-file> [-o=<outputFileName>]
 
 
+DESCRIPTION
+       Note: This is an experimental tool ship under ballerina and will only
```
Comment (MaryamZi) [inline cli/ballerina-tool/src/main/resources/cli-help/ballerina-openapi-gen-client.help]:
```suggestion
       Note: This is an experimental tool which only supports 
```

File: cli/ballerina-tool/src/main/resources/cli-help/ballerina-openapi-gen-client.help [hunk 11]
```diff
@@ -1,12 +1,16 @@
...
+       support limited set of functionality.
```
Comment (MaryamZi) [inline cli/ballerina-tool/src/main/resources/cli-help/ballerina-openapi-gen-client.help]:
```suggestion       
       a limited set of functionality.
```

File: cli/ballerina-tool/src/main/resources/cli-help/ballerina-openapi-gen-client.help [hunk 12]
```diff
@@ -16,16 +20,11 @@ OPTIONS
            This value will be used as the client name when the client is generated.
 
        openapi-file
-           This will be the location of the openapi contract file location which will be
-           used to generate the ballerina client.
+           The openapi contract file location which will be used to generate the
```
Comment (MaryamZi) [inline cli/ballerina-tool/src/main/resources/cli-help/ballerina-openapi-gen-client.help]:
```suggestion
           The location of the OpenAPI contract file which will be used to generate the
```

File: cli/ballerina-tool/src/main/resources/cli-help/ballerina-openapi-gen-client.help [hunk 13]
```diff
@@ -16,16 +20,11 @@ OPTIONS
...
+           ballerina client.
```
Comment (MaryamZi) [inline cli/ballerina-tool/src/main/resources/cli-help/ballerina-openapi-gen-client.help]:
```suggestion
           Ballerina client.
```

File: cli/ballerina-tool/src/main/resources/cli-help/ballerina-openapi-gen-client.help [hunk 14]
```diff
@@ -16,16 +20,11 @@ OPTIONS
...
 
        -o
-           This will be the output location for the generated ballerina client.
-
-
-DEFAULT BEHAVIOR
-       If no options or sub commands are given on execution, Ballerina openapi command will show
-       the help file for the command.
+           Output location for the generated ballerina client.
```
Comment (MaryamZi) [inline cli/ballerina-tool/src/main/resources/cli-help/ballerina-openapi-gen-client.help]:
```suggestion
           Output location for the generated Ballerina client.
```

File: cli/ballerina-tool/src/main/resources/cli-help/ballerina-openapi-gen-contract.help [hunk 15]
```diff
@@ -1,39 +1,42 @@
 NAME
-       Ballerina OpenApi - Gen Contract is a tool which will generate an OpenApi contract for a given Ballerina service.
+       ballerina openapi  gen-contrac - generate an OpenApi contract for a given Ballerina service
```
Comment (MaryamZi) [inline cli/ballerina-tool/src/main/resources/cli-help/ballerina-openapi-gen-contract.help]:
```suggestion
       ballerina openapi  gen-contract - generate an OpenApi contract for a given Ballerina service
```

File: cli/ballerina-tool/src/main/resources/cli-help/ballerina-openapi-gen-contract.help [hunk 16]
```diff
@@ -1,39 +1,42 @@
...
 
-       Note: This is an Experimental tool ship under ballerina hence this will only support limited set of functionality.
 
 SYNOPSIS
-       ballerina openapi gen-contract [<module>:]<service-name> [-i <ballerina-file>] [-o <contract-file>] [-s|--skip-bind]
+       ballerina openapi gen-contract [<module>:]<service-name>
+       [-i <ballerina-file>]
```
Comment (MaryamZi) [inline cli/ballerina-tool/src/main/resources/cli-help/ballerina-openapi-gen-contract.help]: Shall we align L7 and L8 with `[<module>:]<service-name>` in L6?

File: cli/ballerina-tool/src/main/resources/cli-help/ballerina-openapi-gen-contract.help [hunk 17]
```diff
@@ -1,39 +1,42 @@
...
+       [-o <contract-file>] [-s|--skip-bind]
+
+
+DESCRIPTION
+       Note: This is an experimental tool ship under ballerina and will only
```
Comment (MaryamZi) [inline cli/ballerina-tool/src/main/resources/cli-help/ballerina-openapi-gen-contract.help]: Same comment as above regarding experimental status.

File: cli/ballerina-tool/src/main/resources/cli-help/ballerina-openapi-gen-contract.help [hunk 18]
```diff
@@ -1,39 +1,42 @@
...
+       support limited set of functionality.
 
 
 OPTIONS
        module
-           [ Optional if ballerina file is provided ] If value is provided, the contract
-           will be generated by using the given module name.
+           [ Optional if ballerina file is provided ] If provided, the contract
```
Comment (MaryamZi) [inline cli/ballerina-tool/src/main/resources/cli-help/ballerina-openapi-gen-contract.help]:
```suggestion
           [Optional if the Ballerina file is provided] If provided, the contract
```

File: cli/ballerina-tool/src/main/resources/cli-help/ballerina-openapi-gen-contract.help [hunk 19]
```diff
@@ -1,39 +1,42 @@
...
+           will be generated using given module name.
```
Comment (MaryamZi) [inline cli/ballerina-tool/src/main/resources/cli-help/ballerina-openapi-gen-contract.help]:
```suggestion
           will be generated for the specified service in the specified module.
```

File: cli/ballerina-tool/src/main/resources/cli-help/ballerina-openapi-gen-contract.help [hunk 20]
```diff
@@ -1,39 +1,42 @@
...
 
        service-name
-           This option will be used to search the given service name to generate the openapi contract. This
-           option will need either a valid ballerina file or a valid ballerina module.
+           Service name to generate the openapi contract.
```
Comment (MaryamZi) [inline cli/ballerina-tool/src/main/resources/cli-help/ballerina-openapi-gen-contract.help]:
```suggestion
           Name of the service to generate the OpenAPI contract for.
```

File: cli/ballerina-tool/src/main/resources/cli-help/ballerina-openapi-gen-contract.help [hunk 21]
```diff
@@ -1,39 +1,42 @@
...
+           This option will require either a valid ballerina file
```
Comment (MaryamZi) [inline cli/ballerina-tool/src/main/resources/cli-help/ballerina-openapi-gen-contract.help]:
```suggestion
           This option requires either a valid Ballerina file
```

File: cli/ballerina-tool/src/main/resources/cli-help/ballerina-openapi-gen-contract.help [hunk 22]
```diff
@@ -1,39 +1,42 @@
...
+           or a valid ballerina module.
```
Comment (MaryamZi) [inline cli/ballerina-tool/src/main/resources/cli-help/ballerina-openapi-gen-contract.help]:
```suggestion
           or a valid Ballerina module.
```

File: cli/ballerina-tool/src/main/resources/cli-help/ballerina-openapi-gen-contract.help [hunk 23]
```diff
@@ -1,39 +1,42 @@
...
 
-       -i
-           [ Optional if a module is provided ] If value is provided, the contract will be generated using the
-           provided ballerina file and the service name given.
+       -i  <ballerina-file>
+           [ Optional if a module is provided ] If provided,
```
Comment (MaryamZi) [inline cli/ballerina-tool/src/main/resources/cli-help/ballerina-openapi-gen-contract.help]:
```suggestion
           [Optional if a module is provided] If provided,
```

File: cli/ballerina-tool/src/main/resources/cli-help/ballerina-openapi-gen-contract.help [hunk 24]
```diff
@@ -1,39 +1,42 @@
...
+           the contract will be generated using provided ballerina file
+           and the service name.
```
Comment (MaryamZi) [inline cli/ballerina-tool/src/main/resources/cli-help/ballerina-openapi-gen-contract.help]: L28 and L29 

```suggestion
           the contract will be generated for the specified service in the specified file.
```

File: cli/ballerina-tool/src/main/resources/cli-help/ballerina-openapi-gen-contract.help [hunk 25]
```diff
@@ -1,39 +1,42 @@
...
 
-       -o
+       -o  <contract-file>
            This option will specify the output location of the generated contract.
```
Comment (MaryamZi) [inline cli/ballerina-tool/src/main/resources/cli-help/ballerina-openapi-gen-contract.help]:
```suggestion
           The output location of the generated contract.
```

File: cli/ballerina-tool/src/main/resources/cli-help/ballerina-openapi-gen-contract.help [hunk 26]
```diff
@@ -1,39 +1,42 @@
...
 
        -s
            If this option is provided, the generated contract will not be bound to the given ballerina service.
```
Comment (MaryamZi) [inline cli/ballerina-tool/src/main/resources/cli-help/ballerina-openapi-gen-contract.help]: This doesn't seem clear. Not sure if this is still used either.

File: cli/ballerina-tool/src/main/resources/cli-help/ballerina-openapi-gen-contract.help [hunk 27]
```diff
@@ -1,39 +1,42 @@
...
 
 
-DEFAULT BEHAVIOR
-       If no options or sub commands are given on execution, Ballerina openapi command will show
-       the help file for the command.
-
-
 EXAMPLES
-       Get help for Ballerina Gen Contract command
+       Get help for Ballerina  gen-contrac command
```
Comment (MaryamZi) [inline cli/ballerina-tool/src/main/resources/cli-help/ballerina-openapi-gen-contract.help]:
```suggestion
       Get help for Ballerina  gen-contract command
```

File: cli/ballerina-tool/src/main/resources/cli-help/ballerina-test.help [hunk 28]
```diff
@@ -1,105 +1,51 @@
...
+       ballerina test - execute Ballerina tests
 
 SYNOPSIS
-       ballerina test [-lg|--list-groups] [--groups <test_group, ...>]
-           [--disable-groups <test_group, ...>] [--sourceroot <path>]
-           [-e] [--debug <port>] [-c <config_file>]
-           [--exclude-modules <modules, ...>]
-           [--config <config_file>] [-B] [<balfile>|<module-name>]
+       ballerina test [--offline]
+           [--sourceroot <path>]
```
Comment (MaryamZi) [inline cli/ballerina-tool/src/main/resources/cli-help/ballerina-test.help]: Shall we align these with `[--offline]` in L5?

File: cli/ballerina-tool/src/main/resources/cli-help/ballerina-test.help [hunk 29]
```diff
@@ -1,105 +1,51 @@
...
+           [--config <config_file>]
+           [<module-name> | -a | --all]
 
 
 DESCRIPTION
-       Ballerina test command allows you to compile and run Ballerina
-       test sources and prints a summary of test results. You can
-       execute tests in a single test file, a test module, or an
-       entire Ballerina project.
+       Compile and run Ballerina
```
Comment (MaryamZi) [inline cli/ballerina-tool/src/main/resources/cli-help/ballerina-test.help]: Can reformat this for line length 80?

File: cli/ballerina-tool/src/main/resources/cli-help/ballerina-test.help [hunk 30]
```diff
@@ -1,105 +1,51 @@
...
+           When building a module, the remote repository (Ballerina Central) is
+           always checked for latest artifacts of the dependencies. If the
+           '--offline' option is specified, this remote check is not done and
+           the local artifacts will be used instead.
 
        --sourceroot <path>
-           Location of the remote project root with source files. All
-           executions will be relative to the path provided for
-           sourceroot.
+            Location of the remote project root with source files. All
```
Comment (MaryamZi) [inline cli/ballerina-tool/src/main/resources/cli-help/ballerina-test.help]: extra space?

Comment (rdhananjaya) [reply cli/ballerina-tool/src/main/resources/cli-help/ballerina-test.help]: Will send a separate PR with suggested changes

**Pull request merged**